jenkins/entry.sh

```diff
@@ -5,11 +5,17 @@ GID=`id -g`
 
 echo 'jenkins:x:'$GID':' >> /etc/group
 echo 'jenkins:x:'$UID':'$GID':,,,:/home/jenkins:/bin/bash' >> /etc/passwd
+sudo chown -R jenkins:jenkins /home/jenkins
+
+##
+## Start Docker daemon
+##
+nohup dockerd 2>&1 > /dev/null &
```

Comment:
I think we need to mount the `docker` binary and data from the host machines.

Reply:
To isolate environment, I think that using docker in docker without mounting host  dockerd is better. If you have any concerns, please let me know.

Reply:
It's hard for issue tracking if this builder breaks down, and because the images are built inside the container, you need to configure authorization again so that they can be pushed to gcr.io.